db/migrations/postgres/000086_alter_contractapis.up.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+ALTER TABLE contractapis ADD COLUMN message_id UUID NOT NULL;
```

Comment:
This will fail for existing deployments, right? Need to give a default or make it nullable.

Reply:
It looks like this has been missing since the first release that it appeared in: https://github.com/hyperledger/firefly/pull/390/files#diff-b20af3fc418d53f1d729cf53470d90ee03dfeb5ba2e1400650036c120aa107e5